crowdsourcing/viewsets/project.py

```diff
@@ -89,9 +89,9 @@ class ModuleViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         queryset = Module.objects.all()
-        requesterid=self.request.query_params.get('requesterid',None)
```

Comment:
ok ! but don't you think it will cause extra data transmission, while showing list of requesters ? 
Is it possible to select fields in serializer based on request ?